src/client/opamClient.ml

```diff
@@ -1142,14 +1142,18 @@ let install_t t ?ask ?(ignore_conflicts=false) atoms add_to_roots
       OpamConsole.msg "%s"
         (OpamCudf.string_of_conflict t.packages
            (OpamSwitchState.unavailable_reason t) cs);
-      t, Conflicts cs
+      t, if depext_only then None else Some (Conflicts cs)
     | Success solution ->
```

Comment:
I am not sure I understand this change... don't we want to report conflicts normally rather than silently ignore the request if there are any ?

Reply:
Ah indeed! Still don't completely understand why, but seems fine.

Reply:
In case of `depext_only`, we don't need to check the conflicting solution.